crates/sui-core/src/authority/shared_object_version_manager.rs

```diff
@@ -109,18 +249,21 @@ impl SharedObjVerManager {
             );
```

Comment:
this is pedantic, but the word "locking" in "locking shared objects from effects" makes even less sense to me here than it did before now that we aren't even saving the results into a table of any kind

Reply:
Yes, I removed a few of these, good eye.